scripts/dts/extract/globals.py

```diff
@@ -331,6 +331,54 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def child_to_parent_unmap(cell_parent, gpio_index):
+    # Aim of this function is to return a gpio-controller from the
```

Comment:
More of a comment, but instead of "aim of this function...", it might be better to say what the function does.

"This function takes a foo and a bar and returns a (baz, qaz) tuple. If foo can't be found, this other value is returned instead", etc.